Helper/Cart.php

```diff
@@ -2312,6 +2313,7 @@ public function collectDiscounts(
                             $description = $rule->getName();
                         }
                         $discounts[] = [
+                            'rule_id'           => $rule->getRuleId(),
```

Comment:
Based on the code changes I think this field is just used internally by the PHP code. If you send it to Bolt during unmarshal we will discard this data since unmarshal only keeps the fields defined in the struct so it doesnt really change anything in terms of the Bolt code.